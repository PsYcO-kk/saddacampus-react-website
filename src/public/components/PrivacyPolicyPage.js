import React from 'react';

const PrivacyPolicyPage = () => (
	<div>
		<h1>Privacy Policy</h1>
	</div>
);

export default PrivacyPolicyPage;
