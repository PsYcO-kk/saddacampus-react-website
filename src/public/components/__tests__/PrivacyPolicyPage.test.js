import {shallow} from 'enzyme';
import React from 'react';
import PrivacyPolicyPage from '../PrivacyPolicypage'

test("should render PrivacyPolicyPage correctly" , ()=>{
    const wrapper = shallow(<PrivacyPolicyPage />)
    expect(wrapper).toMatchSnapshot();
})
