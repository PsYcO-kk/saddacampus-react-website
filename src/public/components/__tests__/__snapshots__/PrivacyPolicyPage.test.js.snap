// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PrivacyPolicyPage correctly 1`] = `
<div>
  <h1>
    Privacy Policy
  </h1>
</div>
`;
